src/main.rs

```diff
@@ -290,6 +290,34 @@ async fn main() -> Result<()> {
             send_order_id_cmd(&client, &my_key, mostro_key, message, false).await?;
             std::process::exit(0);
         }
+        Some(cli::Commands::Rate {
+            order_id,
+            counterpart_npub,
+            rating,
+        }) => {
+            let mostro_key = XOnlyPublicKey::from_bech32(pubkey)?;
+
+            let vote_content;
```

Comment:
let's change all vote to rate, this way we have consitency with the other Mostro code